net/ghttp/ghttp_request.go

```diff
@@ -224,8 +224,12 @@ func (r *Request) GetRemoteIp() string {
 
 // GetUrl returns current URL of this request.
 func (r *Request) GetUrl() string {
-	scheme := "http"
-	if r.TLS != nil {
+	var (
+		scheme = "http"
+		proto  = r.Header.Get("X-Forwarded-Proto")
+	)
+
+	if r.TLS != nil || (proto != "" && strings.ToLower(proto) == "https") {
```

Comment:
`if r.TLS != nil ||  gstr.Equal(proto, "https") {`

Reply:
You should also update here: https://github.com/gogf/gf/blob/8f326dcac5548d85d465a135b6fa74af70a2be1c/net/ghttp/ghttp_server_log.go#L22

Reply:
Handled